app/src/main/java/com/github/se/polyfit/MainActivity.kt

```diff
@@ -26,7 +27,10 @@ class MainActivity : ComponentActivity() {
         val navigation = Navigation(navController)
         NavHost(navController = navController, startDestination = Route.Register) {
           composable(Route.Register) { LoginScreen(navigation::navigateToHome) }
-          composable(Route.Home) { HomeScreen() }
+          composable(Route.Home) { HomeScreen(navigation) }
+
+          // TODO: Replace with actual data once the API is ready
+          composable(Route.Ingredients) { IngredientScreen(navigation, listOf(), listOf()) }
```

Comment:
I was mostly following the designs on this - i think the idea is that some of the APIs gave a % likelihood of being in the photo so we would take all over a certain % and then recommend ones below it. But, this can change with our actual data so it was just a temporary before the data was setup

Reply:
Okay, Spoonacular provides a list of possible meals along with their associated probabilities. From this, the ingredient list for each meal can be extracted (see: [Spoonacular Food API Documentation](https://spoonacular.com/food-api/docs#Image-Classification-File)). So, how about we could display the ingredients of the top two picks?